### DISCUSSION TITLE
How to show value from relation table ?

### DISCUSSION THREAD

**Discussion by derilkillms** [Laravel-Backpack/community-forum#565] [Q&A (Help)] [answered]
I have two class Users, Roles
field in table users (id,name,roles) roles is foreign_key
field in table roles (id,name)
I want show name from table roles in datatable
please help me

![WhatsApp Image 2023-07-12 at 13 10 49](https://github.com/Laravel-Backpack/community-forum/assets/49135753/ecd932e3-5f66-4d23-9621-5f7d12dcd584)
![WhatsApp Image 2023-07-12 at 13 07 05](https://github.com/Laravel-Backpack/community-forum/assets/49135753/48e45244-921c-4721-ad13-f3c0bf563f0c)

**Comment by o15a3d4l11s2** [comment, marked as answer]
Since you have "roles" (plural) and also you want to add a `select_multiple` field I would suggest that you make `users` table with name, email, password, etc. and a `roles` table with `name` and anything else the role itself needs. Then create a connecting table (pivot table) following this guide: https://laravel.com/docs/10.x/eloquent-relationships#many-to-many it even has an example with users and roles :) Do not forget that you should create the database structure properly and also define the proper relationship between the user and the role models - the first two sections in the link I sent - "Table Structure" and "Model Structure".

Then once you have this, you can use the `select_mutiple` column to show the roles for each user just like you defined it.
